### DISCUSSION TITLE
Wrong Upload Speed Reported by File Browser

### DISCUSSION THREAD

**Discussion by strosz** [filebrowser/filebrowser#3567] [Q&A]
File Browser reports about 20% of the actual upload speed for me when I upload.

At first I was surprised that uploads seemed to complete way earlier than the reported time and got sceptical that the files had actually uploaded completely.

Then I saw it says 6MBs when my network card reported around 30MBs (MByte for both).
So size uploaded and percentage is correct. But speed and time remaining is not.

Is this a bug or anything I need to configure? Much thanks for any help and for the great app.

**Comment by KingLorax** [comment]
I have the same issue. I would also really appreciate it, if someone took a look at it. Maybe it's a bug or just something configured wrong.
